tests/integration-tests/tests/cloudwatch_logging/cloudwatch_logging_boto3_utils.py

```diff
@@ -0,0 +1,88 @@
+import json
+import logging
+
+import boto3
+from botocore.exceptions import ClientError
+
+import utils
+
+LOGGER = logging.getLogger(__name__)
+
+
+def get_log_groups():
+    """Get list of log groups."""
+    logs_client = boto3.client("logs")
+    log_groups = logs_client.describe_log_groups().get("logGroups")
+    LOGGER.debug("Log groups: {0}\n".format(json.dumps(log_groups, indent=4)))
+    return log_groups
+
+
+def get_log_streams(log_group_name):
+    """Get list of log streams."""
+    logs_client = boto3.client("logs")
+    streams = logs_client.describe_log_streams(logGroupName=log_group_name).get("logStreams")
+    LOGGER.debug(
+        "Log streams for {group}:\n{streams}".format(group=log_group_name, streams=json.dumps(streams, indent=4))
+    )
+    return streams
+
+
+def get_log_events(log_group_name, log_stream_name):
+    """Get log events for the given log_stream_name."""
+    logs_client = boto3.client("logs")
+    events = logs_client.get_log_events(logGroupName=log_group_name, logStreamName=log_stream_name).get("events")
```

Comment:
+ minor: we can avoid to define the logs_client
+ What about ClientError? See previous comment

Reply:
The logs_client is defined because `boto3.client("logs").get_log_events(logGroupName...` would be greater than 120 characters in length.

ClientError is ignored by design. The thought is that the existence of the given log stream within the given log group would have already been performed.